string name
string[] labels
geometry_msgs/Point point
float64 metric